### DISCUSSION TITLE
Codeblocks: Padding between line number and code

### DISCUSSION THREAD

**Discussion by ckrenslehner** [welpo/tabi#434] [Q&A] [answered]
Hi!
I have a question: How do you handle the padding between the line numbers and the code?

If I have a code sample with more than 10 line numbers, there is no space between the line number and the code, which looks a little awkward. See this screenshot:

![grafik](https://github.com/user-attachments/assets/c47a7099-fc1b-484c-8459-276b52f9d6f8)

I found that I can add a padding in `tabi\sass\parts\_code.scss` line 27
![grafik](https://github.com/user-attachments/assets/c4dbddf7-84e5-4c2e-911c-c32c5c2c2b3c)

Now it looks like this
![grafik](https://github.com/user-attachments/assets/e4b1b2be-d869-4cb9-a5aa-273bf9c79082)

How can I set this in my page, without editing the theme, or how do you handle such cases?

Thanks for your help! :-)
Regards, Christian

**Comment by welpo** [comment]
That doesn't look good!

Thanks for reporting this. I'll take a look into the styling soon!

**Reply by ckrenslehner** [reply]
Thanks! :-)

**Reply by welpo** [reply, marked as answer]
Just merged #437. Code blocks with line numbers should look much better now! Take a look in the demo: https://welpo.github.io/tabi/blog/markdown/#with-line-numbers

**Comment by ckrenslehner** [comment]
This looks great, thank you! :-)

Óscar ***@***.***> schrieb am Sa., 23. Nov. 2024, 16:50:

**Reply by welpo** [reply]
Just pushed a fix (4be8a5634fa2990825e9bc785029c4dddb340c39) for iOS Safari: code blocks with line numbers were getting huge text sizes and short lines were centered.

Thanks for the report!!
